wce-ote/plugin/plugin.js

```diff
@@ -2690,7 +2696,7 @@
 					isDel=true;
 				}
 
-				if ((originalText && originalText != 'null') || originalText=='') {
+				if ((originalText && originalText != 'null' && originalText != 'undefined') || originalText=='') {
```

Comment:
I realize you didn't write the original logic, and I agree that changing as little as possible right now is a worthy goal, but do you have any idea what good the || clause does on this line?

Reply:
No idea so I left it there. There is quite a difference between the if and the else there and I don't understand all of it at the moment.